src/components/TopBar/components/SearchField/tests/SearchField.test.tsx

```diff
@@ -2,10 +2,7 @@ import React from 'react';
 import {CircleCancelMinor} from '@shopify/polaris-icons';
 import {ReactWrapper} from 'enzyme';
 import {mountWithAppProvider} from 'test-utilities/legacy';
-// We've got to export a named SearchField otherwise other stuff breaks
```

Comment:
Just curious what this was breaking?

Reply:
It complained about being able to generate the types for TopBar.SearchField - or rather it was ok generating them, but trying to use it in consuming projects caused complaints. exporting SearchField in addition to the default component seemed to make it happy.

I don't understand the reasoning for it, but I seemed like it was a due to a combo of a trying to use a withAppProvider wrapped component as an exposed subcomponent

Reply:
It complained about being able to generate the types for TopBar.SearchField - or rather it was ok generating them, but trying to use it in consuming projects caused complaints. exporting SearchField in addition to the default component seemed to make it happy.

I don't understand the reasoning for it, but I seemed like it was a due to a combo of a trying to use a withAppProvider wrapped component as an exposed subcomponent